main.css

```diff
@@ -1,20 +1,64 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400&display=swap');
+
+body {
+  font-family: 'Roboto Slab', serif;
+}
+
 .title {
     text-align: center;
     margin: 2rem;
 }
 
-#about, #timeline, #contact,  #footer{
+#about, #timeline, #cofounder, #footer {
     margin-top: 5rem;
 }
 
 .navbar {
     background-color: #41BBD9;
 }
 
-#home, #about, #timeline, #contact {
-    padding-top: 90px;
+#home, #about, #timeline, #cofounder {
+    padding-top: 80px;
+}
+
+.intro {
+  overflow: hidden;
 }
 
+.home-section-heading {
+  font-size: 5rem;
+  margin-bottom: 0;
+}
+
+.tag-line {
+  font-weight: 400;
+  font-size: 1.6rem;
+}
+
+
+.quote-div {
+  margin: 3.5rem 0;
+}
+
+.quote {
+  margin: 0;
+  background: #41BBD9;
+  padding: 0.8em;
+  border-radius: 1em;
+  color: #fff;
+}
+
+.quote figcaption,
+.quote blockquote {
+  margin: 0.5em;
+}
+
+.quote .links {
+  text-decoration: underline;
+}
+
```

Comment:
Formatting NIT: unneeded spaces, got 3 here instead of 1

Reply:
Done